packages/fluent-editor/src/modules/ai/index.ts

```
@@ -0,0 +1,246 @@
+    }
+    this.sendButtonEl.innerText = text
+    this.sendButtonEl.style.display = isShow ? 'block' : 'none' // 显示发送按钮
+  }
+
+  // 创建AI提示语元素
+  private createAiPreTextEl() {
+    if (!this.aiPreTextEl) {
+      this.aiPreTextEl = document.createElement('span')
+      this.aiPreTextEl.className = 'ql-ai-tip'
+    }
+    this.aiPreTextEl.innerText = `${this.model}帮你写：`
```

ESLint

[error] Prefer `.textContent` over `.innerText`.

(unicorn/prefer-dom-node-text-content)